Dockerfile

```diff
@@ -8,7 +8,11 @@ COPY Cargo.toml .
 RUN cargo build --release
 
 COPY . .
+
+# Force cargo to recompile
+RUN touch -am src/main.rs
```

Comment:
Isn't `touch -am <FILE>` equivalent to `touch <FILE>`?

Reply:
I'll change to `touch <FILE>`. We want to keep compiled libraries, but `cargo clean` removes them.